test/integration/consul-container/test/gateways/gateway_endpoint_test.go

```diff
@@ -179,6 +179,25 @@ func checkHTTPRouteConfigEntry(t *testing.T, client *api.Client, routeName strin
 	}, time.Second*10, time.Second*1)
 }
 
+func checkHTTPRouteConfigEntryWithoutBindStatus(t *testing.T, client *api.Client, routeName string, namespace string) {
```

Comment:
I'd probably update the name to something like, checkConfigEntryExists, but otherwise looks good to me.

Reply:
I like that name a whoooole lot better